New design for alumni page: multiple links & short bio

yechs opened the pull request:
Updating the design for alumni page
It now supports adding multiple links (Personal website / GitHub / LinkedIn) and adding a short bio

IMHO, I think each alumnus/alumna can use the bio for something like "Went to xxx university" / "Working at yyy company now" / "Interested in zzz field" / "Feel free to ask me for ..."
So that current members will know who to contact if they have a specific question in mind

yechs committed:
New Design for alumni page

Add the basic design and looks

yechs committed:
New Design for alumni page: Logic

Make the page logic dynamic:
- auto hides icon if the link is null
- uses favored link for the avatar

yechs committed:
New design for alumni page: fix styles

Fix styles when there are more than one alumni in the same year

yechs commented:
The design should look like this
![image](https://user-images.githubusercontent.com/49149993/78592579-e2d0d680-7877-11ea-9013-e28f2ea1fc08.png)

SamChou19815 reviewed (commented):
The current design doesn't seem to consider the case when bios have varying length.

<img width="1030" alt="Screen Shot 2020-04-06 at 16 56 19" src="https://user-images.githubusercontent.com/4290500/78604298-9a89c800-7827-11ea-9d48-bbc5260db99d.png">

yechs commented:
Oh yeah, I will go change the align-content later

yechs committed:
Alumni page style fix when bio too long

- Stretch the rest of the cards to match the height of the highest
- Inside the stretched cards, stretch the header and bio part to account
for the added space

yechs commented:
I made it look like this

![image](https://user-images.githubusercontent.com/49149993/78644993-6080fa80-78e9-11ea-95b1-2ebadfb22e29.png)

Not ideal but (imho) better than cards with varied heights
